compose.yaml

```
@@ -0,0 +1,12 @@
+services:
+  web:
+    build: .
```

Similarly, we should specify the target here!
```suggestion
    build:
      context: .
      target: development
```